packages/api-client/src/server/electionSetup.ts

```diff
@@ -1,8 +1,42 @@
 import { JointKey } from '../models/jointKey';
 import { getAssignedGuardians } from './guardians';
 import { post } from '../utils/http';
+import { ManifestPreview } from '../models';
+import { SubmitElectionRequest, ValidateManifestRequest } from '../nswag/clients';
 
-export { getManifestPreview } from '../mocks/electionSetup';
+// todo: figure out how to retrieve the current language
+const getLanguage = (): string => 'en';
+
+const getElectionName = (manifest: any): string => {
+    const language = getLanguage();
+    const textList: Array<any> = manifest.name?.text;
+    if (textList) {
+        const languageText = textList.find((i: any) => i.language === language);
+        return languageText.value;
+    }
+    return '';
+};
+
+export const getManifestPreview = (
```

Comment:
This is the heart of this PR.  Not sure if it would have been better sending into the API to parse, but this pattern was already in place, so I followed it.